transform/mattermost-analytics/models/staging/stripe/_stripe__models.yml

```diff
@@ -120,8 +120,12 @@ models:
         description: The license ID for the subscription
       - name: purchase_order_num
         description: The purchase order number for the subscription
+      - name: sfdc_migrated_opportunity_sfid
+        description: The Opportunity SFID of the migrated subscription        
```

Comment:
would help to define/document what a "migrated subscription/opportunity" is.

Reply:
I'm not really sure what a migrated subscription means, I'll talk to John about this.

Reply:
@sri-byte , might want to also ask him to document this in here: https://mattermost.atlassian.net/wiki/spaces/DATAENG/pages/2435874817/Self-serve

Reply:
@sri-byte - see Nick's comment : https://community.mattermost.com/private-core/pl/czpczeuyo7fj3y9qhn4fpt4jty